### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by ChernoAlieu**

### CODE REVIEW

File: Calculator/Calculator.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (TheCSharpAcademy) [inline Calculator/Calculator.csproj:1]: Looks like you didn't include "calculator library" into your repository. Please create a folder with the name chernoAlieu.Calculator and include both projects inside of it. Let me know if you need help. 

image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: Calculator/Calculator.csproj (hunk 1 above)
Comment (ChernoAlieu) [reply Calculator/Calculator.csproj:1]: Should I do that within Visual Studio or from file explorer?

Comment (TheCSharpAcademy) [reply Calculator/Calculator.csproj:1]: Whenever you're dealing with folders I think it's easier to use file explorer

Comment (ChernoAlieu) [reply Calculator/Calculator.csproj:1]: Alright, thank you. Right away then

**Pull request merged**